release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2842,6 +2842,57 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+{product-title} release {product-version}.3, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:8434[RHSA-2024:8434] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:8437[RHSA-2024:8437] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.3 --pullspecs
+----
+
+[id="ocp-4-17-3-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-3-enhancements-exposing-network-overlap-metrics-cno_{context}"]
+===== Exposing network overlap metrics with the Cluster Network Operator
+
+* When you start the limited live migration method and an issue exists with network overlap, the Cluster Network Operator (CNO) can expose network overlap metrics for the issue. This is possible because the `openshift_network_operator_live_migration_blocked` metric now includes the new `NetworkOverlap` label. (link:https://issues.redhat.com/browse/OCPBUGS-39121[*OCPBUGS-39121*])
+
+[id="ocp-4-17-3-automatic-variable-loading-git-repository_{context}"]
+===== Loading git repository environment variables automatically from your repository
+
+* Previously, when you imported a git repository by using the serverless import strategy, the environment variables from the `func.yaml` file were not automatically loaded into the form. With this update, the environment variables are now loaded upon import. (link:https://issues.redhat.com/browse/OCPBUGS-42474[*OCPBUGS-42474*])
+
+[id="ocp-4-17-3-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, a regression was introduced to OpenShift Install during a Power VS deployment. As a result, the security group rules required for OpenShift Install were not created. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-43547[*OCPBUGS-43547*])
+
+* Previously, if an image registry Operator was configured with the`networkAccess` field set to `Internal` in Azure, an authorization error prevented the image registry Operator from deleting the storage container, and the `managementState` field from being set to `Removed`. With this release, the Operator can delete the storage account and storage container, and the `managementState` field can successfully set to `Removed`. (link:https://issues.redhat.com/browse/OCPBUGS-43350[*OCPBUGS-43350*])
+
+* Previously, both active and passive high-availability (HA) deployments ran three replicas instead of the required two. As a result, control planes contain more pods than required which led to scaling issues. With this release, the number of replicas in active and passive HA deployments is reduced from three to two. (link:https://issues.redhat.com/browse/OCPBUGS-42704[*OCPBUGS-42704*])
```

Comment:
```suggestion
* Previously, both active and passive high-availability (HA) deployments ran three replicas instead of the required two. As a result, control planes contained more pods than required, which led to scaling issues. With this release, the number of replicas in active and passive HA deployments is reduced from three to two. (link:https://issues.redhat.com/browse/OCPBUGS-42704[*OCPBUGS-42704*])
```